backup/manager.go

```diff
@@ -241,16 +241,62 @@ func (b *Manager) IsSafeToRunNode(nodeID string) (bool, error) {
 	if err != nil {
 		return false, err
 	}
+
 	for _, s := range snapshots {
 		if s.NodeID == nodeID && s.BackupID != "" && backupID != s.BackupID {
 			b.log.Errorf("remote restore was found for node %v.", nodeID)
 			b.log.Errorf("current backupID=%v, remote backupID-%v", backupID, s.BackupID)
 			return false, nil
+		} else if s.NodeID == nodeID {
```

Comment:
The IsSafeToRunNode is executing multiple time during the app lifecycle. I think in verifying latest backup we want the check to be executed only on startup (once).

Reply:
Moved this logic to a gorutine in the `Start(torConfig *data.TorConfig)` in _app.go_, so now it should only run once on startup.